packages/loaders-compressed-textures/src/index.ts

```diff
@@ -0,0 +1,19 @@
+import { CompressedTextureLoader } from './CompressedTextureLoader';
+import { DDSLoader } from './DDSLoader';
+import { KTXLoader } from './KTXLoader';
+import { Loader } from '@pixi/loaders';
+
+// parse any *.json compressed-textures manifest
+Loader.registerPlugin(CompressedTextureLoader);
```

Comment:
We do not register plugin automatically like this. This should be done in the bundle.